.github/workflows/deploy-storybook.yml

```
@@ -0,0 +1,32 @@
+      run:
+        working-directory: frontend
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 18
+          cache: npm
+          cache-dependency-path: frontend/package-lock.json
+      - run: npm ci
+      - run: npm run build-storybook
+      - uses: peaceiris/actions-gh-pages@v3
```

actionlint (1.7.12)

[error] the runner of "peaceiris/actions-gh-pages@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)